test/unit/vl-mapactions-draw-polygon-action.test.js

```diff
@@ -0,0 +1,30 @@
+import sinon from 'sinon/pkg/sinon-esm';
+import {expect} from 'chai';
+import {Vector as SourceVector} from 'ol/source';
+import {VlDrawPolygonAction} from '../../src/vl-mapactions-draw-polygon-action';
+
+describe('draw polygon action', () => {
+  const source = new SourceVector({});
+
+  const layer = {
+    getSource: () => {
+      return source;
+    },
```

Comment:
idem als al ne keer opgemerkt .. niet mogelijk om hier een echt OL Layer aan te maken? (mockitis/stubitis)

Reply:
Zal verder onderzocht worden in #36.